src/calculateRentalCost.js

```diff
@@ -29,6 +29,23 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const DAY_COST = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUN = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUN = 20;
```

Comment:
I would like you to move those constans outside of the function.

As previous reviewer said, it will allow you to use the values in other functions.